### DISCUSSION TITLE
[Abwesenheitsverwaltung] Abwesenheitsmeldung Schüler

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#17] [User Stories]
### User Story:
Als Schüler möchte ich die Möglichkeit haben meine Abwesenheit selbst online einzutragen, um einfach und schnell die Schule zu informieren.

### Beschreibung:
Es sollte eine eigene Seite geben auf der Schüler sich krankmelden können, anstatt bei der Schule anzurufen. Dabei sollen sie die Dauer und den Grund eintragen können.

**Optional:**
- Oder statt einer eigenen Seite nur ein kleiner Bereich / Button auf einer anderen Seite

### Akzeptanz-Kriterien:
- [ ] Schüler kann sich Krankmelden 
- [ ] Schüler kann die Dauer und den Grund festlegen (Krankheit, Persönlich oder Sportverletzung etc.)
- [ ] Schule wird über die Krankmeldung und die Dauer Informiert

**Comment by laralngn** [comment]
Soll man dort auch eine richtige Krankmeldung hochladen (z.B. vom Arzt oder bei Jüngeren von Eltern unterschriebene Krankmeldung) oder nur so ein Textfeld quasi?

**Reply by Kronprinz03** [reply]
Nein die muss nach wie vor abgegeben werden, oder es läuft online über die Krankenversicherung
